kf/kf_unix.go

```diff
@@ -232,9 +235,48 @@ func (b *BPF) LoadTCAttachProgram(ifaceName, direction string) error {
 		if iface.Name == ifaceName && qdisc.Kind == "clsact" {
 			clsactFound = true
 		}
+		if iface.Name == ifaceName && qdisc.Kind == "htb" {
+			htbFound = true
+			htbHandle = qdisc.Msg.Handle
+		}
+		if iface.Name == ifaceName && qdisc.Kind == "ingress" {
+			ingressFound = true
+			ingressHandle = qdisc.Msg.Handle
+		}
```

Comment:
Can we convert this into switch statement 
```suggestion
		if iface.Name == ifaceName {
		switch qdisc.Kind {
		case "clsact":
			clsactFound = true
		case "htb":
			htbFound = true
			htbHandle = qdisc.Msg.Handle
		case "ingress":
			ingressFound = true
			ingressHandle = qdisc.Msg.Handle
		default:
		          log.Info("qdisc kind for %s : %v", ifaceName, err) 
		}
	}
```

Reply:
Fixed in the newer commit